developer_manual/scripts/theme-bootstrap.sh

```diff
@@ -3,6 +3,7 @@
 # Invoke this script with one argument, the new theme's name.
```

Comment:
``with one argument`` --> ``with two arguments``
see check quantity of arguments below equals to two

Reply:
Thanks for spotting that.